virtool/startup.py

```diff
@@ -55,7 +55,10 @@ def get_scheduler_from_app(app: App) -> aiojobs.Scheduler:
 
 
 async def startup_check_db(app: App):
-    if app["config"].no_check_db:
+
+    config = app["config"]
+
+    if (config.no_check_db if hasattr(config, "no_check_db") else False):
```

Comment:
`no_check_db` should always be set for any startup process that calls this function

Reply:
Some processes in tilt complained `app["config"] has no attribute "no_check_db"`